composer.json

```diff
@@ -9,6 +9,7 @@
         "composer/composer": "^1.0",
         "doctrine/collections": "~1.2",
         "gitonomy/gitlib": "~1.0",
+        "phptype/array": "^1",
```

Comment:
Yes. I wrote it specifically for this PR! It provides the `ArrayType::isMap` used later in this PR.